Earthfile

```diff
@@ -0,0 +1,44 @@
+VERSION 0.6
+FROM rust:1.71
+
+WORKDIR /wolfssl-sys
+
+build-deps:
+    RUN apt-get update -qq
+    RUN apt-get install --no-install-recommends -qq autoconf autotools-dev libtool-bin clang cmake
+    RUN apt-get -y install --no-install-recommends bsdmainutils
+    RUN rustup component add rustfmt
+
+copy-src:
```

Comment:
FYI, I have seen some example in earthly docs to cache dependencies efficiently using cargo-chef:

https://github.com/earthly/earthly/blob/main/examples/rust/Earthfile#L14

We can look into it later too.

Reply:
Yes, I've used cargo chef before. It can be a bit fickle to get it to work reliably, but we should certainly consider it for future work.